[MTM-66665] Improved "Working with dashboards" help text to account for various contexts (#4597)
SoftwareAG/c8y-docs: content/cockpit/working-with-dashboards.md

weight: 40
helpcontent:
  - label: working-with-dashboards
    title: Working with dashboards
    content: "Dashboards let you visualize your data by using a set of widgets. Widgets can display maps, images, graphs, tables, and other graphic representations of data.
    
    
    The availability of the options described below may vary based on the context, for example, whether you are in a home, group or device dashboard.

>     Click **Add dashboard** in the tab bar to add a new dashboard. By default, the dashboard is locked until the edit mode is enabled. To do so, click **Edit widgets** in the top menu bar. Click **Add widget** in the top menu bar to add a new widget to the dashboard. Use the cogwheel icon to edit or remove widgets. Rearrange widgets by drag and drop. Resize widgets using the diagonal arrows icon located in the bottom right corner of the widget.

Reviewer: "To do so" is a bit lost now. To do what?

Author: It refers to enabling the edit mode. Maybe rephrase to:
```suggestion
    Click **Add dashboard** in the tab bar to add a new dashboard. Enable the edit mode by clicking **Edit widgets** in the top menu bar. Then click **Add widget** to add a new widget to the dashboard. Use the cogwheel icon to edit or remove widgets. Rearrange widgets by drag and drop. Resize widgets using the diagonal arrows icon located in the bottom right corner of the widget.
```
?
